ui/admin/app/components/form/field/list-wrapper/text-input/index.hbs

```diff
@@ -0,0 +1,65 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Form::Fieldset
+  @layout={{@layout}}
+  @isRequired={{@isRequired}}
+  @isOptional={{@isOptional}}
+  name={{@name}}
+  as |F|
+>
+
+  {{yield F}}
+
+  <F.Control>
+
+    {{#each @options as |option index|}}
```

Comment:
Should we have been consistent and used the table or changed it for both of them? I think originally we made it a table for semantic meaning, i.e. we're dealing with tabular data. 

I could be convinced it's either a table or unordered list but I don't think we should just have divs.

Reply:
turns out we can't use the segmented group for this scenario, link [here](https://hashicorp.slack.com/archives/C7KTUHNUS/p1698339724997629). For now, I'm following the existing tabular form to keep things simple